introduction.md

```diff
@@ -11,3 +11,7 @@ deadly skills:
 deadly skills
 rebase
 >>>>>>> deadly_skills
+
+*JavaScript
+*Ruby
+*rebase
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?